docs/cloud/dev/spryker-cloud-commerce-os/spryker-cloud-commerce-os-management.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Spryker Cloud Commerce OS management
+description: Assignment of responsibilities, consultation and reporting obligations.
+last_updated: Nov 21, 2022
+template: concept-topic-template
+---
+
+This RACI Matrix defines responsibilities, consultation and reporting obligations while providing Platform-as-a-Service (PaaS) hosting for Spryker customers.
+
+## Glossary
+
+**Responsible**: Those who do the work to complete a task.
+
+**Accountable**: The one ultimately answerable for the correct and thorough completion of a deliverable or task, and the one who delegates the work to those responsible.
+
+**Consulted**: Those whose opinions are sought, typically subject matter experts; and with whom there is a  two-way communication.
+
+**Informed**: Those who are kept up to date on progress, often only on completion of a task or deliverable; and with whom there is just one-way communication.
+
+
+**Application**: Customer Commerce Solution build using Spryker Commerce OS.
+
+**Infrastructure**: Underlying cloud infrastructure managed by Spryker to deliver Spryker Cloud Commerce to Spryker customers.
+
+## RACI Matrix
+
+| ACTIVITY | CUSTOMER | RESPONSIBLE DEVELOPING PARTY (CUSTOMER OR PARTNER) | SPRYKER |
+| --- | ---| --- | --- |
+| Application Health, Security and Performance|
+| Provide code operational application (docker-sdk and Spryker Code) | C | A/R | I | 
+| Optimize Application  performance applying best practices | I/C | R/A | I | 
+| Maintain Application up-to-date and secure | I/C | R/A | I |
+| Infrastructure Configuration and Scaling | 
+| Manage environment sizing depending on the current needs | - | C | R/A |
+| Maintain the Infrastructure up-to-date and secure | I | I | R/A |
+| Configuration of the infrastructure for the Application | I | C | R/A |
+| Provide deploy file that can be used in the Spryker Cloud setup | I | R/A | C |
+| Deployments |
+| Provide valid access tokens for the repositories and the repository is available to Spryker Cloud as per requirements | - | R/A | I |
+| Conduct appropriate (load) testing, especially regarding data imports before production deployment | I/C | R/A | I |
```

Comment:
@michalwroblewski-spryker Is it just load testing or load testing and some other testing?

Reply:
@andriitserkovnyi it is especially load tests but also following industry best practices in testing the solution before deployement